hdf5-vol-daos.spec

```diff
@@ -46,7 +46,7 @@
 
 Name:    hdf5-vol-daos
 Version: %{vol_major}.%{vol_minor}.%{vol_bugrelease}%{?vol_prerelease:~%{vol_prerelease}}
-Release: 8%{?commit:.git%{shortcommit}}%{?dist}
+Release: 9%{?commit:.git%{shortcommit}}%{?dist}
```

Comment:
You don't really need to bump this IMO.  This build should be the same as the existing `-8`, you are just adding new package builds.  You aren't changing the composition of the existing `-8` build in any way.

Reply:
Fair enough, although that's what we did in https://github.com/daos-stack/daos/blob/master/utils/rpms/daos.spec#L675-L676